inst/pages/04_containers.qmd

```diff
@@ -740,3 +740,39 @@ Conversion is possible between other data formats. Interested readers can refer
 * [makeTreeSEFromBiom](https://microbiome.github.io/mia/reference/makeTreeSEFromBiom.html)  
 * [importMetaPhlAn](https://microbiome.github.io/mia/reference/importMetaPhlAn.html)  
 * [importQZA](https://microbiome.github.io/mia/reference/importQIIME2.html)
+
+## Exporting data container {#sec-exporting-data-container}
+
+### Export TreeSummarizedExperiment
+
+Exporting a `TreeSE` data container can be done using *feather* package. TSE object has to be converted into a dataframe (`DataFrame`). The output file is a .feather file, which can be imported in other languages such as Julia or Python. For information, have a look at the [FeatherFile](https://juliapackages.com/p/featherfiles) Julia package and [feather-format](https://pypi.org/project/feather-format/) Python library.
+
+```{r, eval=FALSE}
+# Import TSE
+library(mia)
+data(GlobalPatterns, package="mia")
+tse <- GlobalPatterns
+# Convert into a DataFrame
+tse <- transformAssay(tse, MARGIN = "samples", method="relabundance")
+molten_tse <- mia::meltAssay(tse,
+                        add_row_data = TRUE,
+                        add_col_data = TRUE,
+                        assay.type = "relabundance")
+# Export as a feather file
+library(feather)
+path <- "dirname(rstudioapi::getSourceEditorContext()$path)/../../../tse.feather"
+write_feather(molten_tse,path)
+```
+
+### Export MultiAssayExperiment
+
+Exporting a `MultiAssayExperiment` data container can also be done using *feather* package.
+
+```{r, eval=FALSE}
+# Convert into a DataFrame
+molten_mae <- longFormat(mae)
+molten_mae <- data.frame(molten_mae)
+path <- "dirname(rstudioapi::getSourceEditorContext()$path)/../../../mae.feather"
```

Comment:
You could add just arbitrary path. I added eval=FALSE, so that nothing is really saved

Reply:
like "path/to/mae.feather"